app/javascript/components/pages/FateNew.js

```diff
@@ -70,7 +70,7 @@ class FateNew extends Component {
                 this.state.urls.map((url, keyID) => {
                   return (
                     <div key={keyID}>
-                      <img src={url} className=".h-50px"/>
+                      <img src={url} className=".h-50px" />
```

Comment:
the `.` in `.h-50px` is prolly a typo you dont add the dot for class names like that. (at least not that ive ever seen)

Reply:
Fixed